### DISCUSSION TITLE
Megane 3 - Child lock

### DISCUSSION THREAD

**Discussion by LemurMort** [cedricp/ddt4all#850] [Q&A]
Hello all!

I googled a bit this feature found in BCM.
It seems like it is possible to prevent rear doors from opening from inside of a car until the driver deactivates the button for rear passenger windows.

So I went to BCM95_SW25_V1. 1 - > Doorlock - > Doorlock Configuration and set the CHILD_LOCK_CF to 'true'.
After that a red message appeared on dashboard 'Check child safety device'. The door could be opened from the inside regardless if the button for locking the rear passenger windows was activated.
I set the value back to without.

On russian forum someone mentioned the lock should have 6pin connectors instead of 4pin for this feature to work.

Anyone here knows why the feature doesn't work?
Thanks!

**Comment by Furtif** [comment]
hello if you are looking for answers and not a problem related to the code, please use the [discussions](https://github.com/cedricp/ddt4all/discussions) and close the issue.

**Comment by Furtif** [comment]
done

**Comment by LemurMort** [comment]
Ok, sorry!
And thanks! 🙂

**Reply by Furtif** [reply]
fine

**Comment by Furtif** [comment]
cant not close lol is a discution now!!! close only if as answer

**Reply by LemurMort** [reply]
Ahaaa. I understand, thanks! 🙂

**Comment by Furtif** [comment]
i have moved issue -> discussion

**Comment by AndyIreland** [comment]
I only know of the little lever on the rear doors you slide when they are open , to stop opening the rear doors with the indoor handles - no coding needed

**Reply by LemurMort** [reply]
Hi! I know that too but it's a bit impractical when a grown person sits in the back with kids and wants to squadle out but can't until I go outside and open the door.

**Reply by AndyIreland** [reply]
> Hi! I know that too but it's a bit impractical when a grown person sits in the back with kids and wants to squadle out but can't until I go outside and open the door.

a grown person could reach out of window and open door from outside with the outside door handle?
